dev-env/run-docker-compose.sh

```diff
@@ -65,7 +65,7 @@ function wait_for() {
       return 0
     fi
     echo "Waiting for $service_name to be ready..."
-    sleep 4
+    sleep 10
```

Comment:
That means the function will run for 10 minutes before failing. Do you know which container is particularly slow on your setup? I hoped the latest changes related to the frontend container would have improved the situation.

Reply:
It was the API container. Slow because it was downloading all python packages with a slow network.

I'm fine reverting this change